cli/command/stack/deploy.go

```diff
@@ -19,20 +20,11 @@ func newDeployCommand(dockerCli command.Cli, common *commonOptions) *cobra.Comma
 		Args:    cli.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			opts.Namespace = args[0]
-			switch {
-			case common == nil: // Top level deploy commad
-				return swarm.RunDeploy(dockerCli, opts)
-			case common.orchestrator.HasAll():
-				return errUnsupportedAllOrchestrator
-			case common.orchestrator.HasKubernetes():
-				kli, err := kubernetes.WrapCli(dockerCli, kubernetes.NewOptions(cmd.Flags(), common.orchestrator))
-				if err != nil {
-					return err
-				}
-				return kubernetes.RunDeploy(kli, opts)
-			default:
-				return swarm.RunDeploy(dockerCli, opts)
+			commonOrchestrator := common.orchestrator
+			if common == nil { // Top-level deploy command
```

Comment:
You dereference `common` pointer just the line above 😅

Reply:
and assign to it the line below 🤔